geomstats/learning/pca.py

```diff
@@ -286,6 +286,7 @@ def _fit(self, X, base_point=None):
                 )
 
         # Center data - the mean should be 0 if base_point is the Frechet mean
+        print("hape after", X.shape)
```

Comment:
Typo with "hape".

Reply:
Thanks! Actually this is stray code.  I removed it completely